server/server/settings.py

```diff
@@ -0,0 +1,120 @@
+"""
+Django settings for server project.
+
+Generated by 'django-admin startproject' using Django 3.0.4.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/3.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/3.0/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/3.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = '=sqc-v0#nkjz!cgggyb-n2coxc0_yahmjwoj04zv#ut_v=nka2'
```

Comment:
Secret key should be moved to environment variable, untracked file or so. Maybe we can use GitHub secrets to integrate keys with Heroku deploy? @avan1235 probably knows better.

Reply:
I have to get to know how to use Github secrets in code (not only in workflows commands) but this key is not used while deploying on heroku

Reply:
Okay, let's use local overrides and not worry about deploys now. Here's how Django people do it with `.env` files: [https://djangostars.com/blog/configuring-django-settings-best-practices/#7](https://djangostars.com/blog/configuring-django-settings-best-practices/#7). Extra reading [here](https://12factor.net/config). Let's do the same.

@Rhantolq: since `.env` should be ignored by git, could you add `.env.sample` like they do [here](https://github.com/heroku/identity/blob/master/.env.sample)?